projects/Apps/common/src/index.ts

```diff
@@ -580,6 +580,10 @@ export interface SpecialEdition {
     devUri?: string
     edition: Edition
     expiry: Date
+    header: {
```

Comment:
Required type in `SpecialEdition` to match `RegionalEdition`. This relates to the ticket around changing the title and subtitle on the `IssuePickerHeader`